lib/fix-google-html.js

```diff
@@ -255,6 +255,28 @@ export function unwrapLineBreaks (node) {
   node.children = children;
 }
 
+/**
+ * Moves linebreaks outside of anchor elements,
+ * if the linebreak is the first and/or last child of the anchor.
+ * @param {RehypeNode} node 
+ */
+export function moveLinebreaksOutsideOfAnchors (node) {
+  visit(node, 'element', (node, index, parent) => {
+    console.log(node)
+    if (node.tagName === 'a') {
+      const children = node.children;
+      if (children.length > 1 && children[children.length - 1].tagName === 'br') {
```

Comment:
You can simplify this a bit with optional chaining:

```suggestion
      if (children[children.length - 1]?.tagName === 'br') {
```

Or using `Array.at()` which is in Node 16 (although I’m not totally sure about browser support, but I think it *should* be ok):

```suggestion
      if (children.at(-1)?.tagName === 'br') {
```

Reply:
I opted for the children.length method, not Array.at is a little bleeding edge, and don't want to risk it until eslint-plugin-compat is merged.

Reply:
👍 That sounds reasonable to me.